Convert metric aggs docs runtime fields (#71260)
elastic/elasticsearch: docs/reference/aggregations/metrics/avg-aggregation.asciidoc

==== Script

> It turned out that the exam was way above the level of the students and a grade correction needs to be applied. We can average a <<runtime,runtime field>> to get a corrected average:

Reviewer: ```suggestion
Let's say the exam was exceedingly difficult, and you need to apply a grade correction. Average a <<runtime,runtime field>> to get a corrected average:
```
